LanguageFeatures/Inline-classes/syntax_A02_t03.dart

```diff
@@ -2,54 +2,41 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion A rule for <viewDeclaration> is added to the grammar, along with
-/// some rules for elements used in view declarations:
+/// @assertion A rule for <inlineClassDeclaration> is added to the grammar,
+/// along with some rules for elements used in inline class declarations:
 ///
-/// <viewDeclaration> ::=
-///   'view' 'class' <typeIdentifier> <typeParameters>?
-///       <viewPrimaryConstructor>?
-///       <interfaces>?
+/// <inlineClassDeclaration> ::=
+///   'inline' 'class' <typeIdentifier> <typeParameters>? <interfaces>?
 ///   '{'
-///     (<metadata> <viewMemberDeclaration>)*
+///     (<metadata> <inlineMemberDeclaration>)*
 ///   '}'
 ///
-/// <viewPrimaryConstructor> ::=
-///   '(' <type> <identifier> ')'
+/// <inlineMemberDeclaration> ::= <classMemberDefinition>
 ///
-/// <viewMemberDeclaration> ::=
-///   <classMemberDefinition>
-/// ...
-/// If a view declaration named View includes a <viewPrimaryConstructor> then it
-/// is a compile-time error if the declaration includes a constructor
-/// declaration named View. (But it can still contain other constructors.)
+/// @description Checks that an inline class declaration may contain redirecting
+/// constructors
 ///
-/// @description Checks that it is a compile-time error if a view declaration
-/// named `View` includes a <viewPrimaryConstructor> with named or optional
-/// positional parameter
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=inline-class
 
-view class View1({int id}) {
-//         ^^^^^
-// [analyzer] unspecified
-// [cfe] unspecified
-}
-
-view class View2({required int id}) {
-//               ^
-// [analyzer] unspecified
-// [cfe] unspecified
-}
+import "../../Utils/expect.dart";
 
-view class View3([int id]) {
-//               ^
-// [analyzer] unspecified
-// [cfe] unspecified
+inline class IC {
+  final int id;
```

Comment:
This one _must_ have a plain `IC(this.id);`, otherwise the redirections have no target.

Reply:
Indeed! Thank you